repository/workflow_repository.go

```diff
@@ -178,7 +182,9 @@ func (r *WorkflowRepository) ListWorkflowTemplateVersions(namespace, uid string)
 func (r *WorkflowRepository) ListWorkflowTemplates(namespace string) (workflowTemplateVersions []*model.WorkflowTemplate, err error) {
 	workflowTemplateVersions = []*model.WorkflowTemplate{}
 
-	query, args, err := r.workflowTemplatesSelectBuilder(namespace).ToSql()
+	query, args, err := r.workflowTemplatesSelectBuilder(namespace).
+		Options("DISTINCT ON (wt.id) wt.id,").
```

Comment:
Is that trailing comma okay @rushtehrani ?

Reply:
@Vafilor it's actually needed, [Options](https://godoc.org/github.com/Masterminds/squirrel#SelectBuilder.Options) basically let's you modify the SELECT query manually.

Reply:
@rushtehrani Cool, looking good then.